### PR TITLE
Wrap CONTRIBUTING.md text to 80 columns

### PR DESCRIPTION
**Pull request by strk**
subject says it all

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -56,62 +60,91 @@ ALL TESTS PASSED
...
+I will spend part of my time on gitea
+and lead the development of gitea.
 ```
 
-For a honor to the owners, this document will add the history owners below:
+For a honor to the owners, this document will add the history owners
+below:
 
-2016-11-04 ~ 2017-12-31 lunny <xiaolunwen@gmail.com> tboerger <thomas@webhippie.de> bkcsoft <kim.carlbacker@gmail.com>
+2016-11-04 ~ 2017-12-31
```
Comment (tboerger) [inline CONTRIBUTING.md:132]: Maybe use a list here?

**Comment by tboerger** [comment]
One NITPICK otherwise LGTM

**Comment by strk** [comment]
Used a list, rebased to master and squashed

**Review by tboerger — APPROVED** [review]

**Comment by tboerger** [comment]
LGTM

**Comment by metalmatze** [comment]
LGTM

**Pull request merged**